### PR TITLE
Increases BYOS purchase window

### LINKED ISSUE #15341: Cannot Purchase Build your own Shuttle, haven't met requirements.
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: Upon trying to purchase the build your own shuttle, I was met with the error of "You have not met the requirements for purchasing this shuttle." At the time of purchasing, we had 80k credits, but was unable to call it. Admin is unable to do it either.

 Submitted by: Zer0 (412038981669355560)

### PR DESCRIPTION
**Pull request by redmoogle**
# Document the changes in your pull request

Closes: #15341
20 Minutes -> 30 Minutes
# Wiki Documentation

More time

# Changelog

:cl:  
tweak: You now have 30 minutes to buy the BYOS
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
not a big fan, 40 minutes is super late on high pop chaotic shifts

**Comment by redmoogle** [comment]
> not a big fan, 40 minutes is super late on high pop chaotic shifts

atleast it comes presealed and with air

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -172,20 +172,20 @@
 /datum/map_template/shuttle/emergency/construction
 	suffix = "construction"
 	name = "Build Your Own Shuttle"
-	description = "For the enterprising shuttle engineer! The chassis will dock upon purchase, but launch will have to be authorized as usual via shuttle call. Comes stocked with construction materials."
+	description = "For the enterprising shuttle engineer! The chassis will dock upon purchase, but launch will have to be authorized as usual via shuttle call. Comes stocked with construction materials. Note: Shuttle can not be purchased after 40 minutes."
```
Comment (JamieD1) [inline code/datums/shuttles.dm]:
```suggestion
	description = "For the enterprising shuttle engineer! The chassis will dock upon purchase, but launch will have to be authorized as usual via shuttle call. Comes stocked with construction materials. Note: Shuttle can not be purchased after 30 minutes."
```

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -172,20 +172,20 @@
...
 	admin_notes = "No brig and no medical facilities. Build YOUR own."
 	credit_cost = 5000
 
 /datum/map_template/shuttle/emergency/construction/small
 	suffix = "construction_small"
 	name = "Build Your Own Shuttle, Jr."
-	description = "The full-size BYOS too big for your taste? Aside from the reduced size and cost, this has the all same (lack of) amenities as its full-sized sibling."
+	description = "The full-size BYOS too big for your taste? Aside from the reduced size and cost, this has the all same (lack of) amenities as its full-sized sibling. Note: Shuttle can not be purchased after 40 minutes."
```
Comment (JamieD1) [inline code/datums/shuttles.dm]:
```suggestion
	description = "The full-size BYOS too big for your taste? Aside from the reduced size and cost, this has the all same (lack of) amenities as its full-sized sibling. Note: Shuttle can not be purchased after 30 minutes."
```

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -172,20 +172,20 @@
...
 	admin_notes = "No brig and no medical facilities. Build YOUR own."
 	credit_cost = 2000
 
 /datum/map_template/shuttle/emergency/construction/prerequisites_met()
 	// first 10 minutes only
-	return world.time - SSticker.round_start_time < 12000
+	return world.time - SSticker.round_start_time < 40 MINUTES
```
Comment (JamieD1) [inline code/datums/shuttles.dm]:
```suggestion
	return world.time - SSticker.round_start_time < 30 MINUTES
```

**Comment by JamieD1** [comment]
Changed to 30 mins.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**